src/cli/cli.rs

```diff
@@ -1,5 +1,11 @@
 use clap::{crate_description, crate_name, crate_version, App, AppSettings, Arg, SubCommand};
 
+// pastel_exe is a feature set by build.rs and it's used to avoid the module machinery which
+// doesn't work at build time. Just assume that build.rs is a good friend and that it includes
+// `colorpicker_tools.rs` so that `COLOR_PICKER_TOOLS` are properly defined.
+#[cfg(pastel_exe)]
```

Comment:
Can we rename the feature to `pastel_normal_build`?

For the comment above, I would suggest something like
```
// Only include `colorpicker_tools` for normal builds (not when compiling `build.rs` where
// the module machinery does not work)
```